File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/accessibility/test_helper_accessibility.py
# test___init__.py – Tests helpers accessibility (Python)
import importlib.util
import os

spec = importlib.util.spec_from_file_location(
    "helper_accessibility",
    "/workspaces/dihya.io/backend/components/metiers/threed/fixtures/helpers/accessibility/helper_accessibility.py",
)
helper_accessibility = importlib.util.module_from_spec(spec)
spec.loader.exec_module(helper_accessibility)


def test_is_accessible():
    class Dummy:
        aria_label = "label"

    assert helper_accessibility.is_accessible(Dummy())

File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/accessibility/test_helper_accessibility.js
// test_helper_accessibility.js – Tests helpers accessibility (JS)
const { isAccessible } = require('../../../../fixtures/helpers/accessibility/helper_accessibility');
test('isAccessible', () => {
  expect(isAccessible({ ariaLabel: 'label' })).toBe(true);
  expect(isAccessible({})).toBe(false);
});
